backend/src/variables/index.ts

```diff
@@ -14,6 +14,7 @@ import {
   INTEGRATION_NETLIFY,
   INTEGRATION_GITHUB,
   INTEGRATION_GITLAB,
+  INTEGRATION_GCP_SECRET_MANAGER,
```

Comment:
I'm getting an error here when building Infisical:

`error TS2724: '"./integration"' has no exported member named 'INTEGRATION_GCP_SECRET_MANAGER'. Did you mean 'INTEGRATION_GCP_SECRET_MANAGER_URL'?`

Reply:
Fixed